shared/Microsoft.Extensions.Buffers.MemoryPool.Sources/MemoryPoolSlab.cs

```diff
@@ -38,13 +34,11 @@ public MemoryPoolSlab(byte[] data)
         /// collected and the slab is no longer references the slab will be garbage collected and the memory unpinned will
         /// be unpinned by the slab's Dispose.
         /// </summary>
-        public bool IsActive => _isActive;
-
-        public IntPtr NativePointer => _nativePointer;
+        public bool IsActive { get; private set; }
```

Comment:
Is this ever different than `public bool IsActive => !_isDisposed;`

Reply:
Nope.